inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/workmanager/schedulers/ConfigSchedulerSpec.kt

```diff
@@ -44,4 +79,12 @@ class ConfigSchedulerSpec : BaseTest() {
 
         Mockito.verify(mockConfigScheduler).startConfig()
     }
+
+    @Test
+    fun `should reset delay when max is encountered`() {
```

Comment:
If my calculations are correct. with 60s initial delay, it'll take around 1096000000 years to reach max limit 😄